marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/writerScenario2.spec.tsx

```diff
@@ -386,4 +387,18 @@ describe("Entity Modeling: Writer Role", () => {
     confirmationModal.getDeleteEntityText().should("not.exist");
     graphViewSidePanel.getSelectedEntityHeading("TestEntity").should("not.exist");
   });
+  it("Delete a relationship from graph view", {defaultCommandTimeout: 120000}, () => {
+    // To delete a relation
+    graphVis.getPositionOfEdgeBetween("Person,Patients").then((edgePosition: any) => {
+      cy.waitUntil(() => graphVis.getGraphVisCanvas().click(edgePosition.x, edgePosition.y));
+    });
+    cy.findByTestId("delete-relationship").click();
```

Comment:
Let's create a function under `/support/model/relationship-modal.tsx`, which returns **cy.findByTestId("delete-relationship")** and then just call that function here to fetch the delete icon and perform any action on it.